esp/xslt/esdl2monitor.xslt

```diff
@@ -626,12 +651,24 @@ END;
 </xsl:text>
 </xsl:template>
 
-<xsl:template match="EsdlArray" mode="difference">
+<xsl:template match="EsdlArray" mode="CheckCompare">
+  <xsl:variable name="field"><xsl:call-template name="output_ecl_name"/></xsl:variable>
   <xsl:choose>
+  <xsl:when test="@type='string'">
+      updated_<xsl:value-of select="$field"/> := _df_DiffString<xsl:text>(</xsl:text>
+    <xsl:call-template name="output_active_check">
+      <xsl:with-param name="pathvar">path + '/<xsl:value-of select="$field"/>'</xsl:with-param>
+    </xsl:call-template>
+    <xsl:text>, path + '/</xsl:text><xsl:value-of select="$field"/>/<xsl:call-template name="output_item_tag"/>').AsDataset(L.<xsl:value-of select="$field"/>, R.<xsl:value-of select="$field"/>);
```

Comment:
I'll have to deal with mac systems as a separate issue, I'm not set up to test at the moment.  It would only affect lines with certain characters.  Like this one, if it escapes the &apos;.